site/ru/README.md

```diff
@@ -22,6 +22,19 @@ to the [tensorflow/docs](https://github.com/tensorflow/docs) GitHub repository.
 To volunteer to write or review community translations, contact the
 [docs@tensorflow.org list](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs).
 
+# Как добавить свой перевод?
+
+1. Ознакомьтесь с [уже имеющейся документацией на русском языке](https://github.com/tensorflow/docs/tree/master/site/ru)
+2. Внесите правки в уже готовый перевод или [выберите новый в разделе site/en/](https://github.com/tensorflow/docs/tree/master/site/en)
+3. Переводите на русский язык блоки основные текста документации, а также поясняющие комментарии в коде
+
+Смотрите хороший пример переведенного нами [урока по TensorFlow Keras API здесь](https://github.com/tensorflow/docs/blob/master/site/ru/tutorials/keras/basic_classification.ipynb).
+
+# Советы
+
+1. Не стесняйтесь добавлять не полностью переведенные документации: таким образом другие участники смогут увидеть ваш Pull Request и помочь если требуется
```

Comment:
For incomplete pull requests, GitHub allows you to mark them as *Drafts*. Perhaps mention that and link to here: https://help.github.com/en/articles/about-pull-requests#draft-pull-requests

Reply:
Neat! Adding it, thanks! Brushing up Russian recently? :)